test/Controller/DoiControllerTest.php

```diff
@@ -81,7 +81,7 @@ public function subDoiProvider() : Traversable
         return $this->arrayProvider([
             '10.7554/eLife.00001.001' => '/articles/00001#abstract',
             '10.7554/eLife.00001.002' => '/articles/00001#image1',
-            '10.7554/eLife.00001.003' => '/articles/00001/figures#image1-sd1',
+            '10.7554/eLife.00001.003' => '/articles/00001#image1-sd1',
```

Comment:
Since the source data for the first figure is now in the markup of the page the `lookup/doi` redirect lands on the article page. We need to determine if this is acceptable.